lab-adam/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "lab-adam",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "cowsay": "1.1.8",
+    "querystring": "0.2.0"
```

Comment:
These are both regular dependencies, not dev dependences. Your application will break without them.  You also need your gulp dependencies.